examples/plotting/polar.py

```diff
@@ -1,3 +1,11 @@
+'''This example shows how to use polar coordinates to generate a grid of polar plots of rose/rohdonea curves.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.colors.named, bokeh.io.show, bokeh.models.ColumnDataSource, bokeh.models.Plot, bokeh.models.PolarTransform, bokeh.plotting.figure, bokeh.plotting.figure.line, bokeh.plotting.gridplot
+    :refs: :ref:`ug_topics_contour` > :ref:`ug_topics_contour_polar` 
```

Comment:
```suggestion
    :refs: :ref:`ug_topics_contour` > :ref:`ug_topics_contour_polar`
```

Here is a trailling whitespace. This is the reason why one test failed.